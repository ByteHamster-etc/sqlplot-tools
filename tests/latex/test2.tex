line1
% IMPORTDATA -T test2 test2.data
line2
% PLOT SELECT LOG(2,testsize) AS x, bandwidth AS y FROM test2 WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
line3
% PLOT SELECT LOG(2,testsize) AS x, bandwidth AS y FROM test2
% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
\addplot[red] coordinates { (24,0.144) };
line4