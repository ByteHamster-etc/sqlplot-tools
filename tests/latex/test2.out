line1
% IMPORTDATA -T test2 test2.data
line2
% PLOT SELECT LOG(2,testsize) AS x, bandwidth AS y FROM test2 WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
\addplot coordinates { (10.1699250014423124,20004857425.2276) (11.0874628412503394,20059162540.3994) (11.6438561897747247,20101648154.6247) (12.0443941193584534,20078735758.2602) (12.6147098441152082,20103335011.5598) (13.0223678130284545,19844289277.2422) (13.5999128421871277,19934071922.0195) (14.0112272554232541,19970353548.0849) (14.3309168781146170,19997700036.2933) (14.5924570372680804,20017094792.628) (14.8137811912170371,20024361016.1429) (15.0056245491938781,19588722596.4003) (15.3264294871223031,13352542574.6502) (15.5887146355822637,13352656249.9688) (16.0028150156070540,13353218261.4253) (16.5868397879618265,13353616903.6735) (17.0014081943928084,13353488133.708) (17.5859014496907713,13353703915.8915) (18.0007042690112466,13353783611.6266) (18.5854320515929623,13354375997.764) (19.0003521774803010,13355357557.2646) (19.5851972952600240,13357098956.3436) (20.0001760994864425,13322300787.0644) (20.3220689761960910,12599911132.91) (20.5850799027671085,12597385271.8788) (20.8074555529676222,12593497036.6629) (21.0000880524301222,11814282322.7844) (21.1700032705344726,11732158553.7429) (21.3219985372613852,11151421728.7912) (21.4594956573012618,9635555389.30854) (21.5850212029383591,8830491065.91348) (22.0000440268868273,5703033760.53049) (22.3219633165043094,3764860746.89853) (22.5849918521283265,2970821306.34189) (22.8073800804431672,2625465946.58095) (23.0000220136113603,2448731494.05388) (23.1699445691134424,2352617641.58758) (23.3219457058033224,2288649712.68308) (23.5849771764993851,2222953585.66713) (23.8073675013052261,2195411712.63918) (24.0000110068476675,2180601210.44056) (24.3219369003722143,2170401128.14407) (24.5849698386289317,2168353166.08944) (24.8073612116951253,2167406475.24193) (25.0000055034343306,2167236950.08917) (26.0000027517197896,2167069057.70639) (27.0000013758605508,2167397762.24908) (28.0000006879304394,2167170987.53228) (29.0000003439652607,2167207622.93776) (30.0000001719826406,2167288244.29698) (31.0000000859913229,2167167763.2072) (32.0000000429956621,2167332378.84685) (33.0000000214978312,2167234766.24507) (34.0000000107489156,2166899399.72678) };
line3
% PLOT SELECT LOG(2,testsize) AS x, bandwidth AS y FROM test2
% WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
\addplot[red] coordinates { (10.1699250014423124,21256634665.4957) (11.0874628412503394,21206797610.8078) (11.6438561897747247,21256759869.173) (12.0443941193584534,21256469275.0109) (12.6147098441152082,21254940419.1323) (13.0223678130284545,21033549756.6485) (13.5999128421871277,21105216385.1656) (14.0112272554232541,21142325436.9814) (14.3309168781146170,21164943036.4111) (14.5924570372680804,21177968064.6676) (14.8137811912170371,21188091979.1641) (15.0056245491938781,20167090530.7208) (15.3264294871223031,15434595485.8038) (15.5887146355822637,15445744156.6753) (16.0028150156070540,15470586592.3006) (16.5868397879618265,15476290580.9534) (17.0014081943928084,15478200720.2804) (17.5859014496907713,15452726013.7487) (18.0007042690112466,15403930682.9299) (18.5854320515929623,15424185515.1548) (19.0003521774803010,15423989946.1979) (19.5851972952600240,15426535975.6293) (20.0001760994864425,15416610336.7218) (20.3220689761960910,15270216963.0944) (20.5850799027671085,15269613288.0698) (20.8074555529676222,15269091481.3878) (21.0000880524301222,14380475033.7885) (21.1700032705344726,14310057139.6444) (21.3219985372613852,13667203078.2474) (21.4594956573012618,11857582908.2681) (21.5850212029383591,10975456216.8507) (22.0000440268868273,7462653658.7736) (22.3219633165043094,5239989138.61415) (22.5849918521283265,4381387103.6521) (22.8073800804431672,4033162958.21711) (23.0000220136113603,3847558417.15857) (23.1699445691134424,3743372332.88487) (23.3219457058033224,3666709518.15135) (23.5849771764993851,3581563981.47956) (23.8073675013052261,3543382742.29812) (24.0000110068476675,3522084255.67083) (24.3219369003722143,3507422082.5562) (24.5849698386289317,3503128913.41201) (24.8073612116951253,3502572094.86426) (25.0000055034343306,3501908751.45112) (26.0000027517197896,3502477870.75888) (27.0000013758605508,3502216012.11693) (28.0000006879304394,3502080053.79103) (29.0000003439652607,3502124096.05625) (30.0000001719826406,3501543006.02255) (31.0000000859913229,3501652699.61996) (32.0000000429956621,3501536124.01131) (33.0000000214978312,3501106948.86093) (34.0000000107489156,3500085763.9439) };
line4
